index.html

```diff
@@ -329,5 +325,36 @@ <h2>About me</h2>
         <img class="lastline2" src="images/last line.png" alt="Horizontal-slide">
       </div>
     </section>
+    <section class="modal-section hide" id="modal-section">
+      <div class="modal-contents" id="modal-contents">
+        <!--<div class="title-sect">
+          <h2>Multi Post Stories</h2>
+          <h3>Keeping track of hundreds  of components website</h3>
+          <img src="images/close-icon.png" id="close-projects">
+        </div>
+        <ul class="modal-list-tag">
+          <li class="modal-tags">html</li>
+          <li class="modal-tags">Bootstrap</li>
+          <li class="modal-tags">Ruby on rails</li>
+        </ul>
+        <div class="second-sect">
+          <div class="second-sect-img">
+            <img class="image" src="images/Snapshoot Portfolio.png" alt="Snapshoot image">
+          </div>
+          <div class="last-sect">
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley.
+            </p>
+            <div class="last-buttons">
+              <a href="https://victor-chiemerie.github.io/"><button class="buttons">See Live <img src="images/Live link.png" alt="live_link"></button></a>
+              <a href="https://github.com/Victor-chiemerie/Victor-chiemerie.github.io"><button class="buttons">See Source <img src="images/github.png" alt="github"></button></a>
+            </div>
+          </div>
+        </div>-->
```

Comment:
;- Kindly make sure that the` see the source and see live `buttons are visible in the popup of your desktop version it will make your popup design stick to the Figma design as required in the project :+1: 
- Kindly make sure that in the desktop version your ` see the source and see live ` are inside the modal-contents of the popup it will make your design stick more to the Figma design as required in the project  :+1: 
## Desktop Design 
### Your design

![image](https://user-images.githubusercontent.com/95297251/209113579-3a07334d-7ce5-4e65-a8ac-1b4afc139e5f.png)

### Figma design 
![image](https://user-images.githubusercontent.com/95297251/209113762-c125e5d8-2544-4f1b-8f90-bdd62b91f90a.png)

## Mobile version 

### Your design 
![image](https://user-images.githubusercontent.com/95297251/209115822-6662eb02-9342-4dd3-85db-2229c71ba4cf.png)


### Figma design 
![image](https://user-images.githubusercontent.com/95297251/209115662-8090e7b0-e151-4fca-bfa8-c283244410b4.png)